File: Belajar_CPP/OOP-KelasTerbuka/12- Multi_file/player.cpp

#include "player.h"

// Penjabaran Method
Player::Player(const char* name){
    // Player::name = name;
    this->name = name;
}
void Player::display(){
    cout << "Nama Player: " << this->name << endl;
    cout << "Class ini berada di file terpisah" << endl;
}

string Player::getName(){
    return this->name;
}

void Player::setName(const char* name){
    this->name = name;
}

File: Belajar_Java/_16_HirearkiPackage/com/tutorial/Player.java
package com.tutorial;

import com.terminal.Console;

// visibility deafult
class Player{
    private String name;

    Player(String name){
        this.name = name;
    }

    void setName(String name){
        this.name = name;
    }
    String getName(){
        return this.name;
    }
    void show(){
        // System.out.println("Player name = " + this.name);
        Console.log("Menggunakan Console");
        Console.log("Player name = " + this.name);
    }
}